jobs/mongod/templates/bin/generate_ssl_cert.sh.erb

```diff
@@ -9,14 +9,16 @@ cd "${MONGODB_SSL}" || exit 666
 
 # generate only if client and server keys doesnt already exists
 if [ ! -f "${MONGODB_SSL}/mongodb.pem" ]; then
+    # We need to reset LD Library path to use system deployed openssl version
+    LD_LIBRARY_PATH=""
 
     # retrieving openssl.cnf - location is OS ependent
     OPENSSL_CNF=$(find /etc -name openssl.cnf)
 
     # generate private key
     openssl genrsa -out "${MONGODB_SSL}/mongodb.key" 2048
     
-    SAN_SECTION="[SAN]\nsubjectAltName=IP:<%= spec.ip %>,IP:127.0.0.1\nextendedKeyUsage=serverAuth,clientAuth\n"
+    SAN_SECTION="[SAN]\nsubjectAltName=DNS.1:<%= spec.ip %>,DNS.2:127.0.0.1\nextendedKeyUsage=serverAuth,clientAuth\n"
```

Comment:
We used `IP.1` and `IP.2` with Pierre. It's more consistent to express IP SANs. I wouldn't recommend using `DNS.x` instead.

Reply:
using IPs as subjectAltName provides the following messages in mongodb:

2018-05-30T16:13:26.051+0000 E NETWORK  [thread1] The server certificate does not match the host name. Hostname: 10.165.0.75 does not match SAN(s): 
2018-05-30T16:13:26.053+0000 E NETWORK  [thread1] The server certificate does not match the host name. Hostname: 10.165.0.74 does not match SAN(s): 
2018-05-30T16:13:26.055+0000 E NETWORK  [thread1] The server certificate does not match the host name. Hostname: 10.165.0.73 does not match SAN(s): 

There is no problem when using the DNS instead